### PR TITLE
[HTML] aria-label for Input.Date

### PR DESCRIPTION
**Pull request by quincycs**
Hi,

This is my first pull request on the project :).  I wanted to play with the code a little and see if I could  contribute.  I'd happily accept & implement any feedback.  I signed a CLA awhile ago but I'll let a bot confirm/deny if I need a renewal.

### Summary
Implementation for: (Request: Adding aria-label (speak property) to Input.Date? #2256)

1. I added a 'placeholder' field onto Input.Date.  (side-note, the [json schema](https://github.com/Microsoft/AdaptiveCards/blob/master/schemas/adaptive-card.json#L728) for Input.Date already has this field, but it was never implemented.)
2. The value for this new field is then set on the aria-label.

### Test Plan

I tested the functionality inside the visualizer , designer, and used ChromeVox to also verify that the aria-label was spoken in an expected fashion.

image

I verified that the standalone visualizer updated the HTML to have latest 'placeholder' and 'aria-label' values.  Above screenshot is showing how I tested that.  In addition, I used ChromeVox to verify that a narrator speaks the value in an expected fashion.

image
I verified that the designer now has the additional "placeholder" field and it also updates the json realtime as you type in the field value.

Thanks,
-Quincy Mitchell

### CODE REVIEW

File: source/nodejs/adaptivecards-designer/src/designer-peers.ts [hunk 1]
```diff
@@ -1893,6 +1893,18 @@ export class NumberInputPeer extends InputPeer<Adaptive.NumberInput> {
 }
 
 export class DateInputPeer extends InputPeer<Adaptive.DateInput> {
+    internalAddPropertySheetEntries(card: Adaptive.AdaptiveCard, includeHeader: boolean) {
+        super.internalAddPropertySheetEntries(card, includeHeader);
+
+        let placeholder = addLabelAndInput(card, "Placeholder:", Adaptive.DateInput);
```
Comment (quincycs) [inline source/nodejs/adaptivecards-designer/src/designer-peers.ts]: Whoops, should be `Adaptive.TextInput` here.  Will make change.

**Comment by quincycs** [comment]
Hi, sorry about that, will do!

**Comment by quincycs** [comment]
Should be good now :)

**Comment by paulcam206** [comment]
@dclaux - looks like this never made it in. do you know if it's still needed? if so, can you resolve the conflicts?

**Comment by dclaux** [comment]
Ah darn. I forgot to merge...

The changes are obsolete now, because of a designer refactor. Also, the placeholder property is also missing on Input.Time which this PR doesn't address.

I have opened another issue (https://github.com/microsoft/AdaptiveCards/issues/3410) and will send a separate PR.

**Comment by shalinijoshi19** [comment]
@dclaux, do we want to abandon this PR; If so let's please do it.tx

**Comment by dclaux** [comment]
Closing this. There will be a new PR in the future to address.

**Pull request closed without merging**